src/pages/transactions/transaction-modal.tsx

```diff
@@ -29,7 +29,7 @@ export default function TransactionModal({ userId, close, isOpen }: props) {
       setNewTransaction({
         type: "income",
         concept: "",
-        category: { _id: "", label: "" },
+        category: { _id: "670877bf07255749b8882674", label: "N/A" },
```

Comment:
You could create a constant for this, in any case we need to change this value we do it in the constant and no the  view code